src/js/themes/base.d.ts

```diff
@@ -970,6 +970,9 @@ export interface ThemeType {
   };
   menu?: {
     background?: BackgroundType;
+    drop?: {
+      align?: AlignContentType;
+    };
```

Comment:
Should we be thinking about this a bit more broadly such as a DropType and support other drop props? This would be relevant to this discussion: https://github.com/grommet/grommet/pull/5003#pullrequestreview-598937494

Reply:
I think that would be fair to consider accepting all dropProps (considering they can already be adjusted on Menu via dropProps prop). I had just done align to start because I didn't want to over-expand the theme if unnecessary, but I think given the issue linked, that may be a fair extension of the theme.